content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -214,18 +214,16 @@ kubectl drain <node-to-drain> --ignore-daemonsets
    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
 
    ```shell
-   # replace x in {{< skew currentVersion >}}.x-* with the latest patch version
-   apt-mark unhold kubelet kubectl && \
-   apt-get update && apt-get install -y kubelet='{{< skew currentVersion >}}.x-*' kubectl='{{< skew currentVersion >}}.x-*' && \
-   apt-mark hold kubelet kubectl
+   sudo apt-mark unhold kubelet kubectl && \
+   sudo apt-get update && apt-get install -y kubelet={{< skew currentPatchVersion >}} kubectl={{< skew currentPatchVersion >}} && \
+   sudo apt-mark hold kubelet kubectl
    ```
 
    {{% /tab %}}
    {{% tab name="CentOS, RHEL or Fedora" %}}
 
    ```shell
-   # replace x in {{< skew currentVersion >}}.x-* with the latest patch version
-   yum install -y kubelet-'{{< skew currentVersion >}}.x-*' kubectl-'{{< skew currentVersion >}}.x-*' --disableexcludes=kubernetes
+   sudo yum install -y kubelet-{{< skew currentPatchVersion >}} kubectl-{{< skew currentPatchVersion >}} --disableexcludes=kubernetes
```

Comment:
Does this work (without the `-*`)?

Reply:
I'm waiting on the netifly build to doublecheck.

Reply:
@sftim - so the currentPatchVersion only renders as `1.28.0` not `1.28.1`.  My intent of the `KUBEVER` variable was to follow along with the "Determine which version to upgrade to" and set that as a Variable so its reused in copy+paste steps.

is there another skew variable that I can use to get the `1.28.1` version?

Reply:
That shortcode works how you'd hope; there is a separate problem which is why you don't see the 1.28.1 patch release.

Please file an [issue](https://github.com/kubernetes/kubernetes/issues/new/choose) about the website being stale - https://kubernetes.io/releases/ also doesn't list v1.28.1 (this is because SIG Release haven't updated the release data yet).

Once https://kubernetes.io/releases/ is fixed, you might want to rebase against _main_ from upstream.

As to `KUBEVER`: keep it simple. People who want to write a script can work out how to.